ui-participant/src/hub/HubPage.tsx

```diff
@@ -19,7 +20,8 @@ export default function HubPage() {
   const hubUpdate = useHubUpdate()
   const [showMessage, setShowMessage] = useState(true)
 
-  const unjoinedStudies = portal.portalStudies.filter(pStudy => !userHasJoinedPortalStudy(pStudy, enrollees))
+  const unjoinedStudies = filterUnjoinableStudies(portal.portalStudies)
```

Comment:
"Hub" means "portal", right?
Not for this PR, but if so, are we planning to fix the terminology?